lib/slacker.rb

```diff
@@ -0,0 +1,71 @@
+require "http"
+require "pp"
+class Slacker < LoggingBase
+  class << self
+    def process_slack_conversation(json)
+      event = json["event"]
+      return if event.fetch("subtype", nil) == "bot_message"
+
+      text = event.fetch("text", "")
+      channel = event.fetch("channel", "#talk-big-texan-debug")
+      thread_ts = event.fetch("ts", "")
+
+      if text.downcase == "sweatmeats"
```

Comment:
is this only triggered if the message is exactly `sweatmeats` and not something like `i see sweatmeats`?

Reply:
For testing, I was sending it the word only.  I will move it to a regex.